packages/Babylonian-Printbugger.package/BPSwimlane.class/README.md

```diff
@@ -0,0 +1,5 @@
+A BPSwimlane is a collective, chronologically sorted visualization of all of an example's traced values, effectively displaying all of its probes in an interleaving timeline. It observes its example's trace in order to update itself with added or removed trace values (e.g. a new probe has been added).
+
+Call "visualize" after an example was given through the message "example:" or visualize directly by calling "BPSwimlane newFor:".
+
+The BPPrintbugger was created in the context of the "Live Programming" seminar 2021.
```

Comment:
As we don't have any interactions yet the doc is short rn but will extend in future